helm/alfresco-infrastructure/values.yaml

```diff
@@ -1,15 +1,22 @@
-
 persistence:
   enabled: true
+  baseSize: 20Gi
+  storageClass: #enable and define if you already have a custom storage class defined
+    enabled: false
+    accessModes:
+      - ReadWriteMany
+    name: "nfs" #Custom storage classs name
```

Comment:
Should this be something more generic like `alfresco-sc` to accommodate Azure as well?

Reply:
Storage classes are global so in no case we would force a hardcoded name to use for the storage class. This name here is only as a reference for when you would want to use a custom storage class that you created beforehand.

Reply:
I wasn't suggesting we force a hardcoded name, just that the example could be more generic, maybe `fast-read` or something would have been a better suggestion on my part.  Fine as is though.